File: student_management.py
class Student:
    def __init__(self, id, name, marks):
        self.id = id
        self.name = name
        self.marks = marks

    def display(self):
        print("ID:", self.id)
        print("Name:", self.name)
        print("Marks:", self.marks)


students = []

n = 3

for i in range(n):
    print("Enter details of student", i+1)

    sid = int(input("Enter ID: "))
    name = input("Enter Name: ")
    marks = float(input("Enter Marks: "))

    s = Student(sid, name, marks)
    students.append(s)


print("\nStudent Details")

for s in students:
    s.display()
    print("------------")

File: student_management.c
#include <stdio.h>
#include <string.h>

struct Student
{
    int id;
    char name[50];
    float marks;
};

int main()
{
    struct Student s[3];
    int i;

    printf("Enter details of 3 students\n");

    for(i = 0; i < 3; i++)
    {
        printf("\nStudent %d\n", i+1);

        printf("Enter ID: ");
        scanf("%d", &s[i].id);

        printf("Enter Name: ");
        scanf("%s", s[i].name);

        printf("Enter Marks: ");
        scanf("%f", &s[i].marks);
    }

    printf("\nStudent Details\n");

    for(i = 0; i < 3; i++)
    {
        printf("\nID: %d", s[i].id);
        printf("\nName: %s", s[i].name);
        printf("\nMarks: %.2f\n", s[i].marks);
    }

    return 0;
}